src/livestreamer/plugins/itvplayer.py

```diff
@@ -0,0 +1,183 @@
+import re
+from urlparse import urlparse, urlunparse, urlsplit
```

Comment:
This is an incorrect import on Python 3. Use `livestreamer.compat` instead, though you must also add `urlunparse` and `urlsplit` to that module as they are currently missing.